src/hb-ot-shape-complex-khmer.cc

```diff
@@ -110,13 +110,7 @@ collect_features_khmer (hb_ot_shape_planner_t *plan)
   map->enable_feature (HB_TAG('l','o','c','l'));
   map->enable_feature (HB_TAG('c','c','m','p'));
```

Comment:
I retained how it was before, which IIRC was intentional; ie. I remember in testing we had the `setup_syllables` before `locl` and `ccmp` in USE because I belive Uniscribe had it that way.

Reply:
Oh. I see what you mean. I should add to others as well.

Reply:
Fixed.